src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala

```diff
@@ -211,13 +211,17 @@ object AppConfig {
   }
 
   def readConfig(config: Config): AppConfig = {
-    val googleConfigOption = for {
-      googleServices <- config.getAs[GoogleServicesConfig]("googleServices")
-    } yield GoogleConfig(
-      googleServices,
-      config.as[PetServiceAccountConfig]("petServiceAccount"),
-      config.as[Option[Duration]]("coordinatedAdminSdkBackoffDuration")
-    )
+    val googleEnabled = config.getAs[Boolean]("googleServices.googleEnabled").getOrElse(true)
```

Comment:
Why not just enable/disable Google based on the presence/absence of the `googleServices` config block?

Reply:
because we can't edit sam.conf inside the jar